### PR TITLE
Settings>Roles - field order

### PR DESCRIPTION
**Pull request by shealavington**
This reorders the permissions page to have name and description show above permissions as per the request; https://github.com/directus/api/issues/444

### CODE REVIEW

**Comment by benhaynes** [comment]
Thanks @shealavington! I'll wait for Rijk to review... but this looks good to me!

File: src/lang/locales/nl-NL.js [hunk 1]
```diff
@@ -140,5 +140,7 @@ export default {
   version: "Versie",
   version_and_updates: "Versie en updates",
   view_type: "Bekijk type",
-  yes: "Ja"
+  yes: "Ja",
+  duplicating_field: "Duplicaat veld",
```
Comment (rijkvanzanten) [inline src/lang/locales/nl-NL.js]: Ik denk dat het grammaticaal iets netter is als we zeggen "Dupliceer veld" 😉

Comment (shealavington) [reply src/lang/locales/nl-NL.js]: 😆 Google didn't suggest that, haha!

**Comment by rijkvanzanten** [comment]
> This reorders the permissions page to have name and description show above permissions as per the request

The original request by @benhaynes was:

> Name + Description should be directly below the permissions interface... right now it is below the IP Whitelist.

So I'm afraid this is not fixing the original issue 😬

**Pull request closed without merging**